docs/content.zh/docs/deployment/resource-providers/standalone/docker.md

```diff
@@ -406,15 +406,15 @@ services:
   You can then start creating tables and queries those.
 
 * Note, that all required dependencies (e.g. for connectors) need to be available in the cluster as well as the client.
-  For example, if you would like to use the Kafka Connector create a custom image with the following Dockerfile
+  For example, if you would like to use the Kafka Connector, you can build a custom image.
+  Create a Dockerfile as follows:
 
   ```Dockerfile
   FROM flink:{{< stable >}}{{< version >}}-scala{{< scala_version >}}{{< /stable >}}{{< unstable >}}latest{{< /unstable >}}
   RUN wget -P /opt/flink/lib https://repo.maven.apache.org/maven2/org/apache/flink/flink-sql-connector-kafka_2.12/{{< version >}}/flink-sql-connector-kafka_scala{{< scala_version >}}-{{< version >}}.jar
   ```
 
-  and reference it (e.g via the `build`) command in the Dockerfile.
-  and reference it (e.g via the `build`) command in the Dockerfile.
+  and reference this Dockerfile with the `build` command in the `docker-compose.yml` file.
```

Comment:
Perhaps it better to actually display the `build` command that someone needs to type?

Reply:
Updated the doc. After building the doc screenshot is like:
<img width="1068" alt="Screenshot 2022-10-27 at 10 12 43 PM" src="https://user-images.githubusercontent.com/159186/198507938-d68b6ee6-c642-48cc-8baa-5f125c210109.png">